Podfile

```diff
@@ -15,49 +16,59 @@ def canvas_crashlytics_rn_firebase_pods
+  pod 'React-Core/RCTWebSocket', :path => './rn/Teacher/node_modules/react-native/'
+
+  pod 'React-cxxreact', :path => './rn/Teacher/node_modules/react-native/ReactCommon/cxxreact'
+  pod 'React-jsi', :path => './rn/Teacher/node_modules/react-native/ReactCommon/jsi'
+  pod 'React-jsiexecutor', :path => './rn/Teacher/node_modules/react-native/ReactCommon/jsiexecutor'
+  pod 'React-jsinspector', :path => './rn/Teacher/node_modules/react-native/ReactCommon/jsinspector'
+  pod 'ReactCommon/callinvoker', :path => "./rn/Teacher/node_modules/react-native/ReactCommon"
+  pod 'ReactCommon/turbomodule/core', :path => "./rn/Teacher/node_modules/react-native/ReactCommon"
+  pod 'Yoga', :path => './rn/Teacher/node_modules/react-native/ReactCommon/yoga', :modular_headers => true
+
+  pod 'DoubleConversion', :podspec => './rn/Teacher/node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
+  pod 'glog', :podspec => './rn/Teacher/node_modules/react-native/third-party-podspecs/glog.podspec'
+  pod 'Folly', :podspec => './rn/Teacher/node_modules/react-native/third-party-podspecs/Folly.podspec'
+
+  # node modules
+  # use_native_modules!
+  pod 'BVLinearGradient', :path => './rn/Teacher/node_modules/react-native-linear-gradient'
+  pod 'Interactable', :path => './rn/Teacher/node_modules/react-native-interactable'
+  pod 'react-native-camera', :path => './rn/Teacher/node_modules/react-native-camera'
+  pod 'react-native-document-picker', :path => './rn/Teacher/node_modules/react-native-document-picker'
+  pod 'react-native-image-picker', :path => './rn/Teacher/node_modules/react-native-image-picker'
+  pod 'ReactNativeART', :path => './rn/Teacher/node_modules/@react-native-community/art'
+  pod 'RNAudio', :path => './rn/Teacher/node_modules/react-native-audio'
+  pod 'RNCAsyncStorage', :path => './rn/Teacher/node_modules/@react-native-community/async-storage'
+  pod 'RNDeviceInfo', :path => './rn/Teacher/node_modules/react-native-device-info'
+  pod 'RNFS', :path => './rn/Teacher/node_modules/react-native-fs'
+  pod 'RNSearchBar', :path => './rn/Teacher/node_modules/react-native-search-bar'
+  pod 'RNSound', :path => './rn/Teacher/node_modules/react-native-sound'
+end
+
 abstract_target 'defaults' do
   use_frameworks!
 
-  nm_path = './rn/Teacher/node_modules/'
-  rn_path = nm_path + 'react-native/'
-  pod 'React', :path => rn_path, :subspecs => [
-    'Core',
-    'CxxBridge', # Include this for RN >= 0.47
-    'DevSupport', # Include this to enable In-App Devmenu if RN >= 0.43
-    'RCTText',
-    'RCTNetwork',
-    'RCTWebSocket', # needed for debugging
-    # Add any other subspecs you want to use in your project
-    'ART',
-    'RCTActionSheet',
-    'RCTAnimation',
-    'RCTImage',
-    'RCTSettings',
-    'RCTVibration',
-    'RCTPushNotification',
-    'RCTLinkingIOS',
-    'fishhook'
-  ]
-  # Explicitly include Yoga if you are using RN >= 0.42.0
-  pod "yoga", :path => rn_path + 'ReactCommon/yoga'
```

Comment:
This seems to be our trouble right now, `yoga` -> `Yoga` is making pods delete the Yoga directory sometimes since it "uninstalls" the old `yoga` (that happens to match the same folder in a case-insensitive fs as newly installed `Yoga`)